app/_hub/kong-inc/application-registration/index.md

```diff
@@ -0,0 +1,128 @@
+---
+name: Portal Application Registration
+publisher: Kong Inc.
+version: 1.0.x
+
+desc: Allow portal developers to register applications against services
+description: |
+  Applications allow registered developers on Kong Developer Portal to
+  authenticate with OAuth against a Service on Kong. Admins can
+  selectively admit access to Services using this plugin or
```

Comment:
this plugin and application registration are the same thing.

Reply:
But I agree that we should have docs in plugin section as well.

Reply:
@HenriPro thanks for your feedback. I was clumsily trying to differentiate between gui and curl. Please review again.